docs/index.md

```diff
@@ -166,7 +166,7 @@ are clinically relevant. However, if a private signing key is compromised, then
 
 ### Revocation
 
-Individual Health Cards MAY be revoked using a revocation identifier property `rid` encoded in the `vc` claim of the JWT. This should be a short identifier, meaningless to the verifiers; the only constraint is that the identifier SHALL use the base64url alphabet (but doesn’t need to be base64url encoded) and be no longer than 24 characters. Issuers MAY use application-specific user identifiers for this purpose, but since these could be publicly listed in revocation lists, issuers SHOULD use a one-way transformation of the data combined with enough entropy to prevent reversal. It is RECOMMENDED to use the base64url encoding of the first 64 bits of the output of HMAC-SHA-256 (as specified in [RFC 4868](https://tools.ietf.org/html/rfc4868)) on the user identifier using a 256-bit random secret key concatenated with the `<<kid>>`; i.e., `rid = base64url(hmac-sha-256(secret_key || <<kid>>, user_id)[1..64]).
+Individual Health Cards MAY be revoked using a revocation identifier property `rid` encoded in the `vc` claim of the JWT. This should be a short identifier, meaningless to the verifiers; the only constraint is that the identifier SHALL use the base64url alphabet (but doesn’t need to be base64url encoded, see section 5 of [RFC4648](https://tools.ietf.org/html/rfc4648#section-5)) and be no longer than 24 characters. Issuers MAY use application-specific user identifiers for this purpose, but since these could be publicly listed in revocation lists, issuers SHOULD use a one-way transformation of the data combined with enough entropy to prevent reversal. It is RECOMMENDED to use the base64url encoding of the first 64 bits of the output of HMAC-SHA-256 (as specified in [RFC4868](https://tools.ietf.org/html/rfc4868)) on the user identifier using a 256-bit random secret key concatenated with the `<<kid>>`; i.e., `rid = base64url(hmac-sha-256(secret_key || << kid >>, user_id)[1..64]).
```

Comment:
While we are reviewing this syntax here, is it based on a specific language?

* Double pipe for string concatenation
* Slicing syntax that is  1-indexed, defines spans with `..`, inclusive of upper bound

Reply:
No, I was simply trying to be as human readable as possible.

I just realized I already submitted some fix for the HTML rendering in PR #218. (There are some other md list entries that don't render well.). I should merge the PRs.